changelog/1.17.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "1.17.0"
+description: "Released on 03/17/2021"
+---
+
+### Breaking Changes ❗
+
+- helm: The `namespaceWhitelist` field on the Coder helm charts [has been
+  removed and the organization namespaces feature has been
+  deprecated](../admin/access-control/organizations.md#namespaces).
```

Comment:
**Important**

IMO we shouldn't link to documentation within the changelog, unless the link contains a version. The changelog
should be linked by any other documentation, but since the changelog is immutable, it shouldn't contain links to
pages that may change across versions or be removed.

Reply:
That's a good point -- will remove the links

Reply:
After that, I actually think we're looking ready! Thanks so much :taco: